src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/AzureClientExtensions.cs

```diff
@@ -77,8 +77,6 @@ public static partial class AzureClientExtensions
             IAzureLRO<TBody, THeader> lro = ScheduleLRO<TBody, THeader>(client, response, customHeaders, cancellationToken);
             await lro.BeginLROAsync();
             return await lro.GetLROResults();
-
-            //return await LegacyLro<TBody, THeader>(client, response, customHeaders, cancellationToken);
```

Comment:
@ajesusflores we recently updated our async operation handling code.
The reason the commented code was not removed is to have the legacy code ready to be used as a fallback during debugging.
Yes this is a bad practice and I realized it, but trust me this code is long overdue on cleaning up commented code.
We will clean up this code soon.
Thank you for your diligence.
I am closing this PR by end of this week.